StripeConnect/StripeConnectTests/Internal/Webview/ConnectComponentWebViewTests.swift

```diff
@@ -13,49 +13,127 @@ import WebKit
+                                              componentType: .payouts,
+                                              webLocale: Locale(identifier: "fr_FR"),
+                                              loadContent: false)
+        
+        
+        let result = try await webView.evaluateMessageWithReply(name: "fetchInitParams",
+                                                                json: "{}")
+        
+        guard let json = try? JSONEncoder.connectEncoder.encode(message) else {
+            XCTFail("Failed to encode message \(message)")
+            return
+        }
+        XCTAssertEqual(result, String(data:json, encoding: .utf8))
+    }
+    
+    @MainActor
+    func testUpdateAppearance() async throws {
+        let componentManager = componentManagerAssertingOnFetch()
+        let webView = ConnectComponentWebView(componentManager: componentManager,
+                                              componentType: .payouts,
+                                              webLocale: Locale(identifier: "fr_FR"),
+                                              loadContent: false)
+        var appearance = EmbeddedComponentManager.Appearance()
+        appearance.spacingUnit = 5
+        let expectation = try webView.expectationForMessageReceived(sender: UpdateConnectInstanceSender(payload: .init(locale: "fr-FR", appearance: .init(appearance: appearance, traitCollection: UITraitCollection()))))
+        componentManager.update(appearance: appearance)
+       
+        // Ensures the appearance on component manager was set.
+        XCTAssertEqual(appearance.asDictionary(traitCollection: .init()), componentManager.appearance.asDictionary(traitCollection: .init()))
+        
+        await fulfillment(of: [expectation], timeout: TestHelpers.defaultTimeout)
+    }
+    
+    @MainActor
+    func testFetchInitParamsTraitCollection() async throws {
+        var appearance = EmbeddedComponentManager.Appearance()
+        appearance.colors.actionPrimaryText = UIColor { $0.userInterfaceStyle == .light ? .black : .white }
+
+        let componentManager = componentManagerAssertingOnFetch(appearance: appearance)
+        
+        let webView = ConnectComponentWebView(componentManager: componentManager,
+                                              componentType: .payouts,
+                                              webLocale: Locale(identifier: "fr_FR"),
+                                              loadContent: false)
+
+        webView.triggerTraitCollectionChange(style: .dark)
+        
+        let result = try await webView.evaluateMessageWithReply(name: "fetchInitParams",
+                                                                json: "{}")
+        print(result!)
+        XCTAssertEqual(
+            result,
+            """
+            {"appearance":{"variables":{"actionPrimaryColorText":"\(UIColor.white.cssRgbValue)","fontFamily":"-apple-system"}},"fonts":[],"locale":"fr-FR"}
```

Comment:
optional nit: What do you think about directly using `#FFFFFF` instead of `UIColor.white.cssRgbValue`?

I think it could make this easier to read but it also ensures the color is getting encoded correctly.